src/api/test/functional/search_controller_test.rb

```diff
@@ -635,7 +635,7 @@ def test_find_owner
     assert_xml_tag tag: 'person', attributes: { name: 'king', role: 'maintainer' }
     assert_xml_tag tag: 'group', attributes: { name: 'test_group', role: 'maintainer' }
 
-    # search for not mantainer packages
+    # search for not maintainer packages
```

Comment:
While fixing the typo let's also fix the meaning. 
```suggestion
    # search for un-maintained packages
```